src/test/java/jimmy/mcgymmy/model/date/DateTest.java

```diff
@@ -10,67 +10,116 @@
 
 import jimmy.mcgymmy.commons.exceptions.IllegalValueException;
 
+/**
+ *  "dd-M-yyyy",
+ *  "yyyy/MM/dd",
+ *  "dd/MM/yyyy",
+ *  "dd/M/yyyy",
+ *  "d/M/yyyy",
+ */
+
 class DateTest {
-    private static final String EXPECTED_DATE_STRING_1 = "20 Apr 2020";
-    private static final String EXPECTED_DATE_STRING_2 = "2 Sep 2020";
+    private static final String INVALID_DAY_STRING = "2019-04-31";
+    private static final String INVALID_YEAR_STRING = "-100-12-28";
+    private static final String INVALID_MONTH_STRING = "2019-13-28";
+    private static final String INVALID_DATE_STRING_1 = "31/13/2020";
+    private static final String INVALID_FORMAT_STRING_2 = "20/04/20";
+    private static final String INVALID_FORMAT_STRING_1 = "20//04/2020";
+    private static final String VALID_DATE_STRING_2_D_MMM_YYYY = "2 Sep 2020";
+    private static final String VALID_DATE_STRING_1_YYYY_MM_DD = "2020-04-20";
+
+    private static final String VALID_DATE_STRING_1_DD_MMM_YYYY = "20 Apr 2020";
+
+    private static final String VALID_DATE_STRING_1_SLASH_D_M_YYYY = "2/9/2020";
+    private static final String VALID_DATE_STRING_1_SLASH_D_MM_YYYY = "20/4/2020";
+    private static final String VALID_DATE_STRING_1_SLASH_YYYY_MM_DD = "2020/04/20";
+    private static final String VALID_DATE_STRING_1_SLASH_DD_MM_YYYY = "20/04/2020";
+
+    private static final String VALID_DATE_STRING_1_DASH_D_M_YYYY = "2-9-2020";
+    private static final String VALID_DATE_STRING_1_DASH_YYYY_M_D = "2020-9-2";
+    private static final String VALID_DATE_STRING_1_DASH_DD_MM_YYYY = "20-04-2020";
+    private static final String VALID_DATE_STRING_1_DASH_YYYY_DD_MM = "2020-4-20";
```

Comment:
Variable name does not match string content and MM cannot be 20, just minor naming nit here